query-service-impl/src/main/java/org/hypertrace/core/query/service/pinot/PinotBasedRequestHandler.java

```diff
@@ -430,6 +449,16 @@ public Observable<Row> handleRequest(
     }
   }
 
+  private void measureRequestTimeRangeDuration(Duration timeRangeDuration) {
+    try {
+      long minutes = convertSecondsToMinutes(timeRangeDuration.getSeconds());
```

Comment:
Yes, I was using it before, but I found out later that most of the queries having a span of 60 minutes were being returned as 59 minutes since it wasn't rounding it properly. 
I guess I will use `timeRangeDuration.toMinutes()` again instead of this as now I realize 59 or 60 minutes would be in the same bucket :')